optking/intcosMisc.py

```diff
@@ -11,9 +11,6 @@
 
 # Some of these functions act on an arbitrary list of simple internals,
 # geometry etc. that may or may not be in a molecular system.
-# Also, a few complicated function that act on molecular system
-# forces and Hessians.
-
 
```

Comment:
Will you want to delete this down the road?

Reply:
Maybe working toward that.  Obviously this file is shrinking.  But I think displace() and/or the irc code still uses it so far.